python/lsst/summit/extras/logUtils.py

```diff
@@ -50,7 +55,7 @@ class LogBrowser:
     animal.
 
     example usage:
-    logBrowser = LogBrowser(butler, taskName, collection)
+    logBrowser = LogBrowser(butler, taskName, collection, whereStr=whereStr)
```

Comment:
If we're supporting `where` then I wonder if we should also support passing in a `bind` option too. Don't worry if you don't think it's useful/can't be bothered, and as this is `summit_extras` (and will hopefully be absorbed into `pipetask report` anyway) I don't think it matters much, but just checking, especially given how you're using `.format()` in the example.

Reply:
I wasn't aware of this binding option!  Definitely worth adding, so I did (once I figured out what it was!) 😄

Reply:
There are a few reasons one should always use `bind`, I am told. There is a crazy foot-gun there though, where you mustn't use dimension names as the keys in the dict, which includes `day_obs`, but other than that, it's recommended.

Reply:
Coolio...I added a note about the avoiding foot-gun :)